src/server/node_manager.c

```diff
@@ -1771,7 +1772,7 @@ vnode_unavailable(struct pbsnode *np, int account_vnode)
 				/* If reservation is currently running and a node is down then
 				 * set its substate to degraded
 				 */
-				resv_setResvState(presv, presv->ri_qs.ri_state, RESV_DEGRADED);
+				(void) resv_setResvState(presv, presv->ri_qs.ri_state, RESV_DEGRADED);
```

Comment:
Why void this?  No warning will be emitted.

Reply:
there are 2 other calls to the same function and we coid void the return return value there.